ad3n/ratchet-bundle/0.4/config/packages/ihsan_ratchet.yaml

```diff
@@ -0,0 +1,2 @@
+ihsan_ratchet:
+    web_socket_port: "%env(WEB_SOCKET_PORT)%"
```

Comment:
the manifest should also declare the env variable so that it is added in `.env.dist`

Reply:
done?

Reply:
i just see from SNC Redis, it's ok for you?